### ISSUE TITLE
unexpected interaction with gitlab extension when viewing MR diffs in vscode

### ISSUE THREAD

**Issue by bkcsfi** [sourcery-ai/sourcery#125]
We have an on-premises gitlab version 13.10. I am running code-insiders with extension gitlab.gitlab-workflow version 3.15.1

The gitlab workflow extension allows me to see MR diffs in vscode

When I select a file in the 'current branch MR' to see the diffs, the sourcery output window shows an error

e.g.

```shell
[Error - 12:02:41 PM] Request textDocument/codeAction failed.
  Message: FileNotFoundError: [Errno 2] No such file or directory: '/tplcentral_gateway/tplcentral/processing'
  Code: -32602 
[object Object]
[Error - 12:02:41 PM] Request textDocument/codeAction failed.
  Message: FileNotFoundError: [Errno 2] No such file or directory: '/tplcentral_gateway/tplcentral/processing'
  Code: -32602 
[object Object]
```
Each time I click the 'next diff' arrow, the same error is reported.

I do not expect sourcery to be applied in the diff view. Is that supposed to work? If yes, then maybe its a gitlab extension issue..

![image](https://user-images.githubusercontent.com/6412502/113020074-3ce53e80-9150-11eb-9a8c-618fca30df0e.png)

Sourcery does work when viewing the actual file (instead of the diff)

### Sourcery Version

0.8.8

### Code editor or IDE name and version

Version: 1.55.0-insider
Commit: c185983a683d14c396952dd432459097bc7f757f
Date: 2021-03-30T12:45:45.302Z
Electron: 11.3.0
Chrome: 87.0.4280.141
Node.js: 12.18.3
V8: 8.7.220.31-electron.0
OS: Linux x64 4.8.0-41-generic

### OS name and version

Ubuntu 16.04 

Note that I'm using the 'remote' functionality of vscode, accessing the actual code on another host via ssh

**Comment by Hellebore** [comment]
Thanks for raising this! Definitely an issue on our side - will look into it.

**Comment by Hellebore** [comment]
This should be fixed in the next release, but can't be 100% sure since we don't use Gitlab.

**Comment by Hellebore** [comment]
Should be fixed with release of 0.9.0.
